src/data/set/intervals/basic.lean

```diff
@@ -188,30 +188,30 @@ lemma Ioo_subset_Ioo (h₁ : a₂ ≤ a₁) (h₂ : b₁ ≤ b₂) :
 λ x ⟨hx₁, hx₂⟩, ⟨lt_of_le_of_lt h₁ hx₁, lt_of_lt_of_le hx₂ h₂⟩
 
 lemma Ioo_subset_Ioo_left (h : a₁ ≤ a₂) : Ioo a₂ b ⊆ Ioo a₁ b :=
-Ioo_subset_Ioo h (le_refl _)
+Ioo_subset_Ioo h (le_rfl)
 
 lemma Ioo_subset_Ioo_right (h : b₁ ≤ b₂) : Ioo a b₁ ⊆ Ioo a b₂ :=
-Ioo_subset_Ioo (le_refl _) h
+Ioo_subset_Ioo (le_rfl) h
 
 lemma Ico_subset_Ico (h₁ : a₂ ≤ a₁) (h₂ : b₁ ≤ b₂) :
   Ico a₁ b₁ ⊆ Ico a₂ b₂ :=
 λ x ⟨hx₁, hx₂⟩, ⟨le_trans h₁ hx₁, lt_of_lt_of_le hx₂ h₂⟩
 
 lemma Ico_subset_Ico_left (h : a₁ ≤ a₂) : Ico a₂ b ⊆ Ico a₁ b :=
-Ico_subset_Ico h (le_refl _)
+Ico_subset_Ico h (le_rfl)
 
 lemma Ico_subset_Ico_right (h : b₁ ≤ b₂) : Ico a b₁ ⊆ Ico a b₂ :=
-Ico_subset_Ico (le_refl _) h
+Ico_subset_Ico (le_rfl) h
 
 lemma Icc_subset_Icc (h₁ : a₂ ≤ a₁) (h₂ : b₁ ≤ b₂) :
   Icc a₁ b₁ ⊆ Icc a₂ b₂ :=
 λ x ⟨hx₁, hx₂⟩, ⟨le_trans h₁ hx₁, le_trans hx₂ h₂⟩
 
 lemma Icc_subset_Icc_left (h : a₁ ≤ a₂) : Icc a₂ b ⊆ Icc a₁ b :=
-Icc_subset_Icc h (le_refl _)
+Icc_subset_Icc h (le_rfl)
```

Comment:
This change introduces a lot of noise - can you make a tiny PR that replaces `(le_refl _)` with `le_rfl` and does nothing else?

Reply:
Sure!